subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf52840.h

```diff
@@ -413,8 +413,8 @@ static inline uint32_t hal_radio_phy_mode_get(uint8_t phy, uint8_t flags)
 		mode = RADIO_MODE_MODE_Ble_1Mbit;
 
 #if defined(CONFIG_BT_CTLR_PHY_CODED)
-		/* Workaround: nRF52840 Engineering A Errata ID 164 */
-		*(volatile uint32_t *)0x4000173c &= ~0x80000000;
+		/* Workaround: nRF52840 Revision 3 Errata 191 */
+		*(volatile uint32_t *)0x40001740 &= ~0x80000000;
```

Comment:
How about putting it in `if (nrf52_errata_191())`?

Reply:
done